modules/cms/classes/Controller.php

```diff
@@ -252,6 +252,13 @@ public function run($url = '/')
             $result = $event;
         }
 
+        /*
+         * Set a default samesite config value for invalid values
+         */
+        if (!in_array(strtolower(Config::get('session.same_site', 'lax')), ['lax', 'strict', 'none'])) {
+            Config::set('session.same_site', 'Lax');
+        }
```

Comment:
@ayumi-cloud this should be done in the System service provider

Reply:
@LukeTowers could you write the code location you are thinking and I will move it there, thanks.

Reply:
@ayumi-cloud here: https://github.com/octobercms/october/blob/develop/modules/system/ServiceProvider.php#L99 with whitespace above and below it of course

Reply:
Perfect thanks